.github/workflows/alerting-release-e2e-workflow.yml

```diff
@@ -0,0 +1,17 @@
+name: Alerting Release tests workflow in Bundled OpenSearch Dashboards
+on:
+  pull_request:
+    branches:
+      - main
+      - dev-*
+  push:
+    branches:
+      - main
+      - dev-*
+jobs:
+  tests:
+    uses: ./.github/workflows/release-e2e-workflow-template.yml
+    with:
+      test-name: Alerting
+      test-command: yarn cypress:run-with-security --browser chromium --spec 'cypress/integration/plugins/alerting-dashboards-plugin/*'
+      version: 1.3.0
```

Comment:
Do we need to change this every release?

Reply:
Good suggestion. created issue to read from package.json https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/142